lib/projectLink.js

```diff
@@ -8,6 +8,16 @@ class ProjectLink {
   async save(db) {
     await db.addLink(this.project, this.label, this.url);
   }
+
+  static async findByProject(project, db) {
+    const links = await db.getAllLinksByProject(project);
+    const linksList = [];
+    links.forEach((link) => {
```

Comment:
👏  nice work on this change. This is a great place to use `map` though as well. Give that a try. Shouldn't require any test changes at all.